events-api/src/modules/events/events.controller.ts

```diff
@@ -21,5 +30,82 @@ EventsController.get(
       data: events,
       searchParams,
     });
-  }
+  },
+);
+
+EventsController.get(
+  "/:eventId/tickets",
+  validateEventId(eventsEventIdDtoSchema),
+  async (req, res) => {
+    const eventId = req.params["eventId"]!;
+
+    const tickets = await TicketsService.getTicketsByEventId(eventId);
+    if (!tickets) {
+      return res.status(404).json({
+        message: "Tickets not found",
+      });
+    }
+
+    return res.status(200).json({
+      message: "Tickets retrieved successfully",
+      data: tickets,
+    });
+  },
 );
+
+EventsController.post(
+  "/",
+  validateCreateEventsDto(createEventDtoSchema),
+  async (req, res) => {
+    const requestBody = req.body as unknown as CreateEventDto;
+
+    const result = await EventsService.createEvent(requestBody);
+
+    return res.status(201).json({
+      message: "Events created successfully",
+      data: result,
+    });
+  },
+);
+
+EventsController.delete(
+  "/:eventId",
+  validateDeleteParams(eventsDeleteParamsDtoSchema),
+  async (req, res) => {
+    const eventId = req.params["eventId"];
+    const result = await EventsService.deleteEventById(eventId as string);
+
+    return res.status(204).json({
+      message: result,
+      data: [],
```

Comment:
Please return deleted event data instead of empty array

Reply:
✅